Formula/gobject-introspection.rb

```diff
@@ -6,7 +6,7 @@ class GobjectIntrospection < Formula
   url "https://download.gnome.org/sources/gobject-introspection/1.70/gobject-introspection-1.70.0.tar.xz"
   sha256 "902b4906e3102d17aa2fcb6dad1c19971c70f2a82a159ddc4a94df73a3cafc4a"
   license all_of: ["GPL-2.0-or-later", "LGPL-2.0-or-later", "MIT"]
-  revision 3
+  revision 4
```

Comment:
This patch seems to be only needed at build-time. If so, perhaps we can omit the revision bump.

Reply:
OK, let's try that...